### DISCUSSION TITLE
Mass change of entries

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#38] [Shipped - Live on production]
I often have to plan/replan maintenance for a series of devices. Now I have to open each task individually, make the change to the Due Date and save. It would be great if I could select multiple tasks and then edit all at the same time. Would save me a lot of planning time.

**Comment by YvesR** [comment]
We already can close n-issues, so we look into it how to implement a re-schedule from the grid.
